undefined: reflect.Pointer

**Describe the bug**
# github.com/pelletier/go-toml/v2
D:\go\pkg\mod\github.com\pelletier\go-toml\v2@v2.0.4\unmarshaler.go:1143:37: undefined: reflect.Pointer
D:\go\pkg\mod\github.com\pelletier\go-toml\v2@v2.0.4\unmarshaler.go:1211:20: undefined: reflect.Pointer

**To Reproduce**
go get -u all

**Expected behavior**
dont report error when do "go get -u "

**Versions**
 - go-toml: v2.0.4
 - go: v1.17.11
 - operating system: Windows

**Additional context**
Add any other context about the problem here that you think may help to diagnose.

Same issue when we create a docker image.
 # github.com/pelletier/go-toml/v2
#32 287.9 ../../../../pkg/mod/github.com/pelletier/go-toml/v2@v2.0.4/unmarshaler.go:1143:37: undefined: reflect.Pointer
#32 287.9 ../../../../pkg/mod/github.com/pelletier/go-toml/v2@v2.0.4/unmarshaler.go:1211:20: undefined: reflect.Pointer

--- a/unmarshaler.go
+++ b/unmarshaler.go
@@ -1140,7 +1140,7 @@ func fieldByIndex(v reflect.Value, path []int) reflect.Value {
 	for i, x := range path {
 		v = v.Field(x)
 
-		if i < len(path)-1 && v.Kind() == reflect.Pointer {
+		if i < len(path)-1 && v.Kind() == reflect.Ptr {
 			if v.IsNil() {
 				v.Set(reflect.New(v.Type().Elem()))
 			}
@@ -1208,7 +1208,7 @@ func forEachField(t reflect.Type, path []int, do func(name string, path []int)) 
 
 		if f.Anonymous && name == "" {
 			t2 := f.Type
-			if t2.Kind() == reflect.Pointer {
+			if t2.Kind() == reflect.Ptr {
 				t2 = t2.Elem()
 			}
 

